### DISCUSSION TITLE
Channel description instead of channel number for key presses in automations

### DISCUSSION THREAD

**Discussion by eikowagenknecht** [SukramJ/aiohomematic#168] [Ideas]
I noticed that with the new way to handle key presses in automations via dropdown (which is great btw!) there is an opporinity to make it even clearer which device is meant.

CCU3:
![image](https://user-images.githubusercontent.com/1475672/149139451-b9257415-0812-4961-a42b-a561e2ac2b0b.png)

Home Assistant:
![image](https://user-images.githubusercontent.com/1475672/149139503-d8ccafee-beeb-4dd6-a1f8-0a8ddcdf641f.png)

Is it possible to use the channel name instead of the channel number in that part of the UI?

Like this:
- Taster "EG.Büro Taster Eingang" wurde gedrückt
- Taster "EG.Büro Taster Eingang" wurde lange gedrückt
- ...

**Comment by SukramJ** [comment]
I used the channel name when starting implementing this, but then i realized that this is problematic:

1. When you change the channel name the automation will  break.
2. This may cause problems with translations.
3. My main reason: **That will not work with blueprints**

There is something like a `title` missing for triggers and action to make better readable in UI, that is not part of the required key.

**Reply by eikowagenknecht** [reply]
Ah, of course on the technical side it needs to be stored as "1, 2, 3, ..." to make it robust against renaming etc.

So HA core needs to add a way to display a custom title for actions first? That seems like a valid feature request for the core devs :-)

**Reply by SukramJ** [reply]
This is maybe your second feature request for HA.
